Maple2.Server.Game/Packets/ChatStickerPacket.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using Maple2.Model.Game;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class ChatStickerPacket {
+    private enum Command : byte {
+        Load = 0,
+        ExpiredNotify = 1,
+        Add = 2,
+        Use = 3,
+        GroupChat = 4,
+        Favorite = 5,
+        Unfavorite = 6
+    }
+
+    public static ByteWriter Load(List<int> favorites, List<ChatSticker> stickers) {
```

Comment:
I generally prefer `ICollection<>` here rather than specific implementations (`List<>`). You can still pass lists to `ICollection<>`.